### ISSUE TITLE
dcm files can be found when running preprocessing pipeline

### ISSUE THREAD

**Issue by brucehoff** [Sage-Bionetworks/DigitalMammographyChallenge#348]
https://www.synapse.org/#!Synapse:syn4224222/discussion/threadId=1348

**Comment by tschaffter** [comment]
Same as #347

**Comment by tschaffter** [comment]
Closing because duplicate of #347.

**Comment by tschaffter** [comment]
Yuanfang Guan is still complaining about this issue.

```
hello i encountered this error again:
/trainingData/146747.dcm not found,
this is submission id: 7862222
i found the missing images are very different on every machine, where the program dies in such an unpredictable way, can you please check?
```

What machine did she use?

**Comment by brucehoff** [comment]
@tschaffter Thomas:
The referenced submission (7862222) ran on bm16.

**Comment by tschaffter** [comment]
Did that job run before or after we decommissioned bm16?

**Comment by brucehoff** [comment]
It ran for a few minutes this morning.  Should bm16 be off line?

**Comment by tschaffter** [comment]
Yes because it's missing training images. As suggested today, I'll configure it to run scoring submissions.

**Comment by brucehoff** [comment]
@tschaffter Mea culpa.  I must have accidentally restarted the agents on bm16.  I just took the following actions:
- I verified that there are no running jobs on bm16.
- I stopped the agents.
- I updated the script "updateTrainAgents.sh" to omit bm16.  (I think this is the cause of the problem:  I probably updated all the agents, inadvertently restarting bm16.)
- I checked to see what submissions have run on bm16 in the last few days (since this issue was opened).  They are:
7862222 (the latest one reported by Yuangfang)
7859568 (run yesterday by another user, who has since run other submissions, always on other machines)
